massgen/cli/input.py

```
@@ -0,0 +1,446 @@
+        first_line = input(prompt).strip()
+    except RuntimeError:
+        # No running loop - safe to use sync prompt
+        try:
+            session = _get_prompt_session()
+            if not enable_path_completion:
+                first_line = session.prompt(prompt, completer=None).strip()
+            else:
+                first_line = session.prompt(prompt).strip()
+        except (EOFError, KeyboardInterrupt):
+            raise
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)